src/tokens/mainnet.json

```diff
@@ -2376,5 +2376,13 @@
     "symbol": "PUFFER",
     "decimals": 18,
     "logoURI": "https://assets.coingecko.com/coins/images/50630/large/puffer.jpg?1728545297"
+  },
+  {
+    "chainId": 1,
+    "address": "0xe343167631d89B6Ffc58B88d6b7fB0228795491D",
+    "name": "Global Dollar",
+    "symbol": "USDG",
+    "decimals": 18,
```

Comment:
i believe decimals for this token should be 6:
https://etherscan.io/token/0xe343167631d89B6Ffc58B88d6b7fB0228795491D
<img width="413" alt="image" src="https://github.com/user-attachments/assets/5f00b27a-1ec1-43a0-9472-2a293c7a6c49">